examples/timeline/other/drag&drop.html

```diff
@@ -171,7 +171,6 @@
     event.dataTransfer.setData("text", JSON.stringify(objectItem));
   }
 
-
   var items = document.querySelectorAll('.items .item');
 
```

Comment:
OK, never mind. I feel silly about commenting on 'too many empty lines' anyway. There's probably a reason why you want it like this.